internal/service/dns_config/model_config_host.go

```diff
@@ -0,0 +1,209 @@
+
+	"github.com/infobloxopen/terraform-provider-bloxone/internal/flex"
+)
+
+type ConfigHostModel struct {
+	AbsoluteName         types.String `tfsdk:"absolute_name"`
+	Address              types.String `tfsdk:"address"`
+	AnycastAddresses     types.List   `tfsdk:"anycast_addresses"`
+	AssociatedServer     types.Object `tfsdk:"associated_server"`
+	Comment              types.String `tfsdk:"comment"`
+	CurrentVersion       types.String `tfsdk:"current_version"`
+	Dfp                  types.Bool   `tfsdk:"dfp"`
+	DfpService           types.String `tfsdk:"dfp_service"`
+	Id                   types.String `tfsdk:"id"`
+	InheritanceSources   types.Object `tfsdk:"inheritance_sources"`
+	KerberosKeys         types.List   `tfsdk:"kerberos_keys"`
+	Name                 types.String `tfsdk:"name"`
+	Ophid                types.String `tfsdk:"ophid"`
+	ProtocolAbsoluteName types.String `tfsdk:"protocol_absolute_name"`
+	ProviderId           types.String `tfsdk:"provider_id"`
+	Server               types.String `tfsdk:"server"`
+	SiteId               types.String `tfsdk:"site_id"`
+	Tags                 types.Map    `tfsdk:"tags"`
+	Type                 types.String `tfsdk:"type"`
+}
+
+var ConfigHostAttrTypes = map[string]attr.Type{
+	"absolute_name":          types.StringType,
+	"address":                types.StringType,
+	"anycast_addresses":      types.ListType{ElemType: types.StringType},
+	"associated_server":      types.ObjectType{AttrTypes: ConfigHostAssociatedServerAttrTypes},
+	"comment":                types.StringType,
+	"current_version":        types.StringType,
+	"dfp":                    types.BoolType,
+	"dfp_service":            types.StringType,
+	"id":                     types.StringType,
+	"inheritance_sources":    types.ObjectType{AttrTypes: ConfigHostInheritanceAttrTypes},
+	"kerberos_keys":          types.ListType{ElemType: types.ObjectType{AttrTypes: ConfigKerberosKeyAttrTypes}},
+	"name":                   types.StringType,
+	"ophid":                  types.StringType,
+	"protocol_absolute_name": types.StringType,
+	"provider_id":            types.StringType,
+	"server":                 types.StringType,
+	"site_id":                types.StringType,
+	"tags":                   types.MapType{ElemType: types.StringType},
+	"type":                   types.StringType,
+}
+
+var ConfigHostResourceSchemaAttributes = map[string]schema.Attribute{
+	"absolute_name": schema.StringAttribute{
+		Computed:            true,
+		MarkdownDescription: "Host FQDN.",
+	},
+	"address": schema.StringAttribute{
+		Computed:            true,
+		MarkdownDescription: "Host's primary IP Address.",
+	},
+	"anycast_addresses": schema.ListAttribute{
+		ElementType:         types.StringType,
+		Computed:            true,
+		MarkdownDescription: "Anycast address configured to the host. Order is not significant.",
+	},
+	"associated_server": schema.SingleNestedAttribute{
+		Attributes: ConfigHostAssociatedServerResourceSchemaAttributes,
+		Optional:   true,
+	},
```

Comment:
Why is this field missing the Markdown description?

Reply:
This should be the description: https://github.com/Infoblox-CTO/ddi.dns.config/blob/master/api/swagger-spec/objects.swagger.json#L3714

Reply:
added